pinecone/utils/convert_to_list.py

```diff
@@ -0,0 +1,9 @@
+def convert_to_list(obj):
+    class_name = obj.__class__.__name__
+
+    if class_name == 'list':
+        return obj
+    elif hasattr(obj, 'tolist'):
+        return obj.tolist()
+    else:
+        return list(obj)
```

Comment:
All classes that implement `__iter__` methods can be passed to the builtin python `list()` method, but calling `list()` on a numpy ndarray gives back a list of `int64` objects which is numpy-specific and causes the type checking in `Vector` to blow up. 

Numpy's `Ndarray` and pandas' `IntegerArray`, `Serial`, etc have an alternative method called `tolist()` that returns a list of native python `int`s instead. So in this function we just check if that method is present and call it if it is. This lets us to the conversion without creating an explicit dependency like before in places where we were checking if something was a numpy object with `isinstance`.